metaci/release/migrations/0014_auto_20210406_0241.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.1.7 on 2021-04-06 02:41
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def add_legacy_change_case_template(apps, schema_editor):
+    ChangeCaseTemplate = apps.get_model("release", "ChangeCaseTemplate")
+    Release = apps.get_model("release", "Release")
+    case_template = ChangeCaseTemplate(name="Legacy Release", case_template_id=1)
+    case_template.save()
+    Release.objects.filter(change_case_template__isnull=True).update(
+        change_case_template=case_template
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("release", "0013_implementationstep"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="release",
+            name="change_case_template",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.PROTECT,
+                to="release.changecasetemplate",
+            ),
+        ),
+    ]
```

Comment:
lines 23-32 are completely unused here, because you are replacing the `operations` list on line 34